### PR TITLE
Codeowners: Add file

### PR DESCRIPTION
**Pull request by valentinpalkovic**
Closes N/A

## What I did

Created a CODEOWNERS file

## Checklist

- [ ] Make sure your changes are tested (stories and/or unit, integration, or end-to-end tests)
- [ ] Make sure to add/update documentation regarding your changes
- [ ] If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

- [ ] When this PR is ready for testing, make sure to add `ci:normal`, `ci:merged` or `ci:daily` GH label to it to run a specific set of sandboxes. The particular set of sandboxes can be found in `code/lib/cli/src/sandbox-templates.ts`
- [ ] Make sure this PR contains **one** of the labels below.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "build", "documentation", "maintenance", "dependencies", "other"]`

### CODE REVIEW

**Review by ndelangen — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Code Owners
+
+# Scripts
+/scripts/ @ndelangen @kasperpeulen
```
Comment (JReinhold) [inline CODEOWNERS:4]:
```suggestion
/scripts/ @ndelangen @kasperpeulen
/scripts/release/ @jreinhold
```

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# Code Owners
+
```
Comment (JReinhold) [inline CODEOWNERS:2]:
```suggestion

# Root
.github/ @JReinhold @yannbf @vanessayuenn
.circleci/ @yannbf @kasperpeulen
.yarnrc.yml @ndelangen @JReinhold

# Docs
docs/ @kylegach @jonniebigodes

```

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+# Scripts
+/scripts/ @ndelangen @kasperpeulen
+
+# Addons
+/code/addons/a11y/                 @ndelangen @yannbf
+/code/addons/actions/              @ndelangen @yannbf
+/code/addons/backgrounds/          @yannbf @ndelangen
+/code/addons/controls/             @kasperpeulen @ndelangen
+/code/addons/docs/                 @kylegach @kasperpeulen
```
Comment (JReinhold) [inline CODEOWNERS:11]:
```suggestion
/code/addons/docs/                 @JReinhold @kasperpeulen
```

File: CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+/code/addons/outline/              @yannbf @valentinpalkovic
+/code/addons/storyshots-core/      @ndelangen
+/code/addons/storyshots-puppeteer/ @ndelangen
+/code/addons/storysource/          @ndelangen
+/code/addons/toolbars/             @ndelangen @kylegach
+/code/addons/viewport/             @yannbf @ndelangen
+
+# Builder
+/code/builders/builder-manager/  @ndelangen @valentinpalkovic
+/code/builders/builder-vite/     @ndelangen @valentinpalkovic @IanVS
```
Comment (JReinhold) [inline CODEOWNERS:28]:
```suggestion
/code/builders/builder-vite/     @JReinhold @valentinpalkovic @IanVS
```

Based on https://chromaticqa.slack.com/archives/C04S3F04Q68/p1687909632587579?thread_ts=1687877773.854549&cid=C04S3F04Q68

File: CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+/code/builders/builder-webpack5/ @ndelangen @valentinpalkovic
+
+# Frameworks
+/code/frameworks/angular/                 @valentinpalkovic @yannbf
+/code/frameworks/html-vite/               @kasperpeulen @kylegach
+/code/frameworks/html-webpack5/           @kasperpeulen @kylegach
```
Comment (JReinhold) [inline CODEOWNERS:34]:
```suggestion
/code/frameworks/html-vite/               @kasperpeulen @JReinhold 
/code/frameworks/html-webpack5/           @kasperpeulen @JReinhold 
```

File: CODEOWNERS [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+/code/addons/actions/              @ndelangen @yannbf
+/code/addons/backgrounds/          @yannbf @ndelangen
+/code/addons/controls/             @kasperpeulen @ndelangen
+/code/addons/docs/                 @kylegach @kasperpeulen
+/code/addons/essentials/           @valentinpalkovic @ndelangen
+/code/addons/gfm/                  @ndelangen @valentinpalkovic
+/code/addons/highlight/            @yannbf @valentinpalkovic
+/code/addons/interactions/         @yannbf @ndelangen
+/code/addons/jest/                 @ndelangen
+/code/addons/links/                @yannbf @kylegach
```
Comment (JReinhold) [inline CODEOWNERS:17]:
```suggestion
/code/addons/links/                @yannbf @JReinhold
```

File: CODEOWNERS [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+/code/addons/measure/              @yannbf @valentinpalkovic
+/code/addons/outline/              @yannbf @valentinpalkovic
+/code/addons/storyshots-core/      @ndelangen
+/code/addons/storyshots-puppeteer/ @ndelangen
+/code/addons/storysource/          @ndelangen
+/code/addons/toolbars/             @ndelangen @kylegach
```
Comment (JReinhold) [inline CODEOWNERS:23]:
```suggestion
/code/addons/toolbars/             @ndelangen @JReinhold
```

File: CODEOWNERS [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+/code/renderers/react/          @valentinpalkovic @kasperpeulen
+/code/renderers/server/         @shilman @valentinpalkovic
+/code/renderers/svelte/         @JReinhold @kasperpeulen
+/code/renderers/vue3/           @kasperpeulen @JReinhold
+/code/renderers/web-components/ @kasperpeulen @JReinhold
+
+# UI
+/code/ui/blocks/     @JReinhold @cdedreuille
+/code/ui/components/ @cdedreuille @JReinhold
+/code/ui/manager/    @ndelangen @JReinhold @cdedreuille
```
Comment (JReinhold) [inline CODEOWNERS:85]:
```suggestion
# UI
/code/ui/.storybook/ @JReinhold @cdedreuille
/code/ui/blocks/     @JReinhold @cdedreuille
/code/ui/components/ @cdedreuille @JReinhold
/code/ui/manager/    @ndelangen @JReinhold @cdedreuille
```

**Review by JReinhold — CHANGES_REQUESTED** [review]
I see @kylegach mentioned a couple of places where I was suppose to be I think, maybe a faulty search-and-replace @valentinpalkovic ?

File: CODEOWNERS [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+# Lib
+/code/lib/channels/       @ndelangen @kasperpeulen
+/code/lib/cli/            @yannbf @valentinpalkovic @ndelangen
+/code/lib/cli-sb/         @yannbf @valentinpalkovic @ndelangen
+/code/lib/cli-storybook/  @yannbf @valentinpalkovic @ndelangen
+/code/lib/client-logger/  @ndelangen @yannbf
+/code/lib/codemod/        @kasperpeulen @ndelangen
+/code/lib/core-common/    @ndelangen @yannbf
+/code/lib/core-events/    @ndelangen @kasperpeulen
+/code/lib/core-server/    @ndelangen @JReinhold
```
Comment (tmeasday) [inline CODEOWNERS:55]:
```suggestion
/code/lib/core-server/    @ndelangen @JReinhold @tmeasday @shilman 
```

(Indexer stuff)

File: CODEOWNERS (hunk 8 above)
Comment (tmeasday) [reply CODEOWNERS:85]: I think I should be added to `/code/ui/manager` too there are enough bits of that in the APIs I am responsible for.

**Review by yannbf — APPROVED** [review]
Looks pretty good to me!

**Pull request merged**